src/index.ts

```diff
@@ -252,6 +264,16 @@ export async function newEventStore<Q>(
     });
   });
 
+  emitter.subscribe('_eventstore.EventReplayRequested', async (event: Event<EventReplayRequested, EventContext<any>>) => {
```

Comment:
can we extract the callback function and use the same one in both subscriptions?